src/app/client/src/app/modules/public/module/course/components/public-course/public-course.component.ts

```diff
@@ -132,6 +132,7 @@ export class PublicCourseComponent implements OnInit, OnDestroy, AfterViewInit {
     const option = {
       source: 'web',
       name: 'AnonymousCourse',
+      organisationId: this.hashTagId || '*',
```

Comment:
is this.hashTagId or  this.userService.hashTagId,

In the leaner-page.component.ts you are using 
 this.userService.hashTagId

Reply:
it is non-logged user. hence userserive.hastagid will not be present.